scenario/tasks.yaml

```diff
@@ -53,13 +53,13 @@ tasks:
    act_trans_diff_file:  ['TaskTransfer', 'SSHFileToFile', 'info', 'instances', 'diff']
    act_trans_diff_file_chunks:  ['TaskTransfer', 'SSHChunksTransfer', 'info', 'instances', 'diff']
    act_is_not_copy_diff_file: ['IsNotCopyDiffFile']
-   act_get_info_inst_src: ['GetInfoInstances', 'src_cloud']
-   act_get_info_images_src: ['GetInfoImages', 'src_cloud']
-   act_get_info_volumes_src: ['GetInfoVolumes', 'src_cloud']
+   act_get_info_inst_src: ['GetInfoInstancesList', 'src_cloud']
+   act_get_info_images_src: ['GetInfoImagesList', 'src_cloud']
+   act_get_info_volumes_src: ['GetInfoVolumesList', 'src_cloud']
    act_get_info_objects_src: ['GetInfoObjects', 'src_cloud']
-   act_get_info_images_dst: ['GetInfoImages', 'dst_cloud']
-   act_get_info_inst_dst: ['GetInfoInstances', 'dst_cloud']
-   act_get_info_volumes_dst: ['GetInfoVolumes', 'dst_cloud']
+   act_get_info_images_dst: ['GetInfoImagesList', 'dst_cloud']
+   act_get_info_inst_dst: ['GetInfoInstancesList', 'dst_cloud']
+   act_get_info_volumes_dst: ['GetInfoVolumesList', 'dst_cloud']
```

Comment:
Let's also rename class names, because they are somewhat confusing: `GetInfoVolumesList`, for example, does not return anything, so there's no actual "get" there ;)